### PR TITLE
[FIX] Leaderboard Table Issue Solved

### LINKED ISSUE #179: [Bug] Leaderboard table Issue
### Describe the bug
Leaderboard table too short to div.

### To Reproduce
1. Go to https://ajay-dhangar.github.io/algo/leaderboard
2. See error

### Expected behavior
Leaderboard table should be full width with the div.

### Screenshots
image

### Please add hactoberfest and GSSoC24 tag. You can assign me for this issue.

---

**Would you like to work on this issue?**  
- [x] Yes  
- [ ] No

### PR DESCRIPTION
**Pull request by 4F24L**
### Description
This pull request fixes the alignment issue of the leaderboard table in the React application. The table is now centered, improving visual presentation and user experience.

- Fixes #179

### Before : 
image

### After ✅ : 
image

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -87,4 +87,16 @@ a:hover {
...
+  width: 100%; /* Ensure the background div is full width */
+}
+
+.table-auto {
+  width: 100%; /* Ensure the table is full width */
+}
+
+.container {
+  max-width: none; /* Ensure the container does not restrict width */
+}
```
Comment (ajay-dhangar) [inline src/css/custom.css]: I suggest deleting the CSS code you added. We are using Tailwind CSS, so you can apply its class names instead.

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Please update your code based on the suggestions I've provided. Also, please explain why you are pushing the `package-lock.json` file. If you don't have a good reason, please consider removing it.

File: src/css/custom.css (hunk 1 above)
Comment (4F24L) [reply src/css/custom.css]: Okay...I can make changes with class names without writing css.

I will fix it soon.

**Comment by 4F24L** [comment]
@ajay-dhangar review it !!

**Comment by ajay-dhangar** [comment]
> @ajay-dhangar review it !!

@4F24L, you need to sync your code with the latest changes in the main branch. To do this, you can use `git pull origin main`. Once your code is up to date, the error should be resolved.

**Review by 4F24L — COMMENTED** [review]
Changes made

**Review by ajay-dhangar — APPROVED** [review]
Fine

**Pull request merged**